### PR TITLE
PR:Modal

### PR DESCRIPTION
**Pull request by Jayesh3636**
Added Modal

Hey guys I have added Modal designs!
Can anyone please review it!
let me know the changes if required!
TIA~!

https://user-images.githubusercontent.com/91378157/158057852-e92cc670-6839-42b3-8691-7b5b4e539097.mp4

### CODE REVIEW

File: Components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (devdeadviz) [inline Components/modal/modal.html:7]: You can use custom meta info to give it personal touch. ` <meta name="author" content="Jayesh Patil">`

File: Components/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        <nav class="nav">
+            <div class="nav__left">
+                <a href="../../Index.html">
+                    <div class="nav__logo">
+                        <img src="../../images/DARKBUCKET-title.svg" class="nav__logo">
+                    </div>
+                </a>
+            </div>
+            <div class="nav__right">
+                <div class="social-media-list">
```
Comment (devdeadviz) [inline Components/modal/modal.html:36]: Use the same type of naming notation.

File: Components/modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+
+                        <!-- <basic input starts> -->
+
+                        <p class="para-rg"> Copy the code below and you're ready to go.<br><br> </p>
+
+                        <h5>Offer Modal</h5>
+                        <div class="example-container vm">
+                            <div class="modal">
+                                <div class="modal-content-nm vm display-flex-items">
```
Comment (devdeadviz) [inline Components/modal/modal.html:142]: What si nm and vm?
Add proper comments to describe them to other users.

File: reset.css [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 :root {
     --primary-color: #2780f2;
+    --light-primary-color: #93b0ff;
+    --dark-primary-color: #081527;
     --hover-primary-color: #1968ce;
     --secondary-color: rgb(219, 219, 219);
     --hover-secondary-color: rgb(165, 165, 165);
```
Comment (devdeadviz) [inline reset.css:9]: Use the same color value notation.

**Review by devdeadviz — APPROVED** [review]
Just dropped some suggestions,, have a look at it.

**Review by pavan6115 — APPROVED** [review]

**Pull request merged**